### PR TITLE
[Feat - ADF 736]: Incorporate in items the metadata properties restrictions

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-736

### CODE REVIEW

File: views/cypress/support/manageSchema.js [hunk 1]
```diff
@@ -44,3 +46,26 @@
     cy.get('button[type="submit"]').click();
     cy.wait('@editClass');
 });
+
+Cypress.Commands.add('findInputInManageSchema', (options) => {
```
Comment (clagosarias) [inline views/cypress/support/manageSchema.js:50]: missing JSDoc

Comment (habemuscode) [reply views/cypress/support/manageSchema.js:60]: Done

File: views/cypress/support/manageSchema.js [hunk 2]
```diff
@@ -16,6 +16,8 @@
  * Copyright (c) 2021 (original work) Open Assessment Technologies SA ;
  */
 
+import selectors from "../../../../taoItems/views/cypress/utils/selectors";
```
Comment (clagosarias) [inline views/cypress/support/manageSchema.js:19]: this is not being used I believe

**Review by clagosarias — APPROVED** [review]

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
lgtm

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**